src/components/CopyToClipBoard/CopyToClipBoard.test.tsx

```diff
@@ -1,28 +1,27 @@
-import { ReactWrapper } from 'enzyme';
 import React from 'react';
 
 import { copyToClipBoardUtility } from 'verdaccio-ui/utils/cli-utils';
-import { mount } from 'verdaccio-ui/utils/test-enzyme';
+import { render, cleanup, fireEvent } from 'verdaccio-ui/utils/test-react-testing-library';
 
 import CopyToClipBoard from './CopyToClipBoard';
-import { CopyIcon } from './styles';
 
 jest.mock('verdaccio-ui/utils/cli-utils');
 
 describe('<CopyToClipBoard /> component', () => {
-  let wrapper: ReactWrapper;
+  let wrapper: any;
   const copyText = 'copy text';
 
   beforeEach(() => {
-    wrapper = mount(<CopyToClipBoard text={copyText} />);
+    cleanup();
```

Comment:
Testing Library says that we should not perform manual cleanup because of its own internal mechanism, except some cases. Is this one of the corner cases??

Reply:
Only if you enable in the setup, not yet enabled, we need to review all test first.